wix-bookings/guides/About Wix Bookings.md

```diff
@@ -4,42 +4,58 @@
 
 Some examples of Wix Bookings use cases are:
 
-- Beauty & wellness services: e.g., book a haircut or a massage
-- Fitness services: e.g., book a yoga class or a private training session
-- Education services: e.g., book a programming course or a personal math tutor
+- **Beauty & wellness services:** e.g., book a haircut or a massage
+- **Fitness services:** e.g., book a yoga class or a private training session
+- **Education services:** e.g., book a programming course or a personal math tutor
 
-Learn more about about how site collaborators can [create and manage their bookings]. (https://support.wix.com/en/article/wix-bookings-about-wix-bookings)
+Learn more how site collaborators can [create and manage their bookings](https://support.wix.com/en/article/wix-bookings-about-wix-bookings). 
 
 With the Wix Bookings APIs, you can:
```

Comment:
I assume this was here from before, but this section doesn't read well. I suggest changing this sentence to something like 
Wix Booking APIs include: